alot/commands/globals.py

```diff
@@ -978,14 +978,15 @@ def _set_base_attributes(self):
             self.envelope.tags = [t for t in self.tags.split(',') if t]
 
     async def _set_subject(self, ui):
+        subject = ""
```

Comment:
Perhaps this should not be hard-coded but a default taken from the config, which defaults to something non-empty such as "[no subject]"?